app/connectors/CustomsDeclareExportsMovementsConnector.scala

```diff
@@ -85,10 +86,6 @@ class CustomsDeclareExportsMovementsConnector @Inject()(appConfig: AppConfig, ht
   def fetchQueryNotifications(conversationId: String, eori: String)(implicit hc: HeaderCarrier): Future[HttpResponse] =
     httpClient
       .GET[HttpResponse](s"${appConfig.customsDeclareExportsMovements}$IleQuery/$conversationId", eoriQueryParam(eori))
-      .recoverWith {
-        case exception: Upstream4xxResponse if exception.upstreamResponseCode == Status.FAILED_DEPENDENCY =>
-          Future.successful(HttpResponse(responseStatus = Status.FAILED_DEPENDENCY))
-      }
       .andThen {
```

Comment:
Recover with was not useful, it just for specific http status from response return a new response with the same status.

The whole logic for it is in Controller and handle everything only based on the status